services/mempool/mempool.go

```diff
@@ -164,6 +164,7 @@ func (mp *TxPool) addTransaction(utxoView *blockchain.UtxoViewpoint,
 		},
 		StartingPriority: CalcPriority(msgTx, utxoView, height, mp.cfg.BD),
```

Comment:
I wonder why not to modify the TxDesc definition directly? You want a priority for every tx when you first see it & add it to the mempool, it's hurt nothing to define at TxDesc
```golang
type TxDesc struct {
	// Tx is the transaction associated with the entry.
	Tx *Tx

	// Added is the time when the entry was added to the source pool.
	Added time.Time

	// Height is the block height when the entry was added to the the source
	// pool.
	Height int64

	// Fee is the total fee the transaction associated with the entry pays.
	Fee int64

	// FeePerKB is the fee the transaction pays in meer per 1000 bytes.
	FeePerKB int64

	// StartingPriority is the priority of the transaction when it was added
	// to the pool.
	StartingPriority float64
}
```